domains/felipe.json

```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+      "username": "felipecalderon",
+      "email": "hola@felipe.bio"
+    },
+    "repo": "https://github.com/felipecalderon",
+    "record": {
+      "URL": "https://felipe.bio/"

```

Comment:
You're requesting vercel verification record yet using a URL, double check your records per the docs: https://docs.is-a.dev